hack/tools/kapinger/clients/http.go

```diff
@@ -41,7 +41,7 @@ func NewKapingerHTTPClient(clientset *kubernetes.Clientset, labelselector string
 			Transport: &http.Transport{
 				DisableKeepAlives: true,
 			},
-			Timeout: 3 * time.Second,
+			Timeout: 15 * time.Second,
```

Comment:
What if there's a networkpolicy dropping traffic in our testing? This would reduce that amount of traffic by 80%?

Reply:
perhaps make this timeout configurable?